src/utils/batt_utils.ts

```
@@ -0,0 +1,231 @@
+      const osascriptCmd = `/usr/bin/osascript -e 'do shell script "${battCmd} status > ${tempOutputFile} 2>&1"'`;
+      
+      // Execute the command via osascript
+      execSync(osascriptCmd, { encoding: 'utf8' });
+      
+      // Read the output from the temporary file
+      const fs = require('fs');
+      if (fs.existsSync(tempOutputFile)) {
+        const output = fs.readFileSync(tempOutputFile, 'utf8').trim();
+        
+        // Clean up the temporary file
+        try { fs.unlinkSync(tempOutputFile); } catch (e) { /* ignore cleanup errors */ }
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)